flow-typed/npm/babel-core_vx.x.x.js

```diff
@@ -0,0 +1,234 @@
+// flow-typed signature: 833ebc885d74656d211747e19241c7f1
+// flow-typed version: <<STUB>>/babel-core_v^6.26.3/flow_v0.80.0
+
```

Comment:
Yea, what I'm wondering is two things -- do we even want any of these checked in? And if we do, how can we make updating them as easy as `flow-typed update`?

Reply:
Well, it'd probably be easiest to run `flow-typed update` with all of the auto-generated template files checked in, otherwise, we'd have to remove those files each time we updated the definitions.

Reply:
Yea, I ran `flow-typed update` and this was the result. Any thoughts on the first question -- why even check these in at all?

Reply:
That's the convention they propose. So we should keep them checked in. Just need to run `flow-typed update/install --skip` to skip installing auto-generated templates.